### DISCUSSION TITLE
Add more examples in the nadb protocol documentation

### DISCUSSION THREAD

**Discussion by mathemagie** [nabaztag2018/pynab#302] [Ideas]
for example in the document https://github.com/nabaztag2018/pynab/blob/master/PROTOCOL.md, is it possible to add an example of a json command (like this one {"type":"ears", "left": 10, "right": 15}) to send to the nabd TCP/IP server for each type of packet sent : stage, info, ears, command, message, etc 

Other request : to write a file (to add in the github of course) which contains all the possible commands in json to send to Nabd.

thank you very much

**Comment by f-laurens** [comment]
Repository [pynab_cli](https://github.com/f-laurens/pynab_cli) contains, in its `json` subdirectory, a number of example JSON command files, that may help practice with the various packets that may be sent to nabd.

**Reply by mathemagie** [reply]
this is exactly what I wanted :) thanks a lot
